kivy/core/window/__init__.py

```diff
@@ -1388,6 +1388,19 @@ def to_widget(self, x, y, initial=True, relative=False):
     def to_window(self, x, y, initial=True, relative=False):
         return (x, y)
 
+    def to_normalized_pos(self, x, y):
+        '''Transforms absolute to normalized coordinate using
```

Comment:
```suggestion
        '''Transforms absolute pos to normalized (0-1) coordinates using
```

Reply:
Maybe it's better to use "coordinates" instead of "pos" because that wording is used in `Widget.to_parent` and `Widget.to_local`?
Therefore:  "Transforms absolute coordinates to normalized (0-1) coordinates using :attr:`system_size`."

Reply:
yeah, although `to_norm_coord` is probably better!? Rather than having to type it all out and spell it correctly...

Reply:
I would rather use `to_normalized_pos`.

Reply:
That's OK with me.